com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -268,10 +268,7 @@ private void Cleanup()
                 m_NetworkAnimatorStateChangeHandler = null;
             }
 
-            if (IsServer)
-            {
-                NetworkManager.OnClientConnectedCallback -= OnClientConnectedCallback;
-            }
+            NetworkManager.OnClientConnectedCallback -= OnClientConnectedCallback;
```

Comment:
can we have this change only in isolation or does changes in testproject are also needed?